### ISSUE TITLE
Ошибка запуска ютуба

### ISSUE THREAD

**Issue by RuVl** [Priler/jarvis#78]
![image](https://user-images.githubusercontent.com/78366536/234985006-724a2a7f-57d5-4505-b721-d63185d1eb3c.png)
Браузер может запустить хром, но не запускает ютуб, т.к. в хроме несколько пользователей. При выполнении команды возникает ошибка (скриншот), а не выбор пользователя хром

**Comment by rim41Dev** [comment]
Хром скорее всего лежит по другому пути. Попробуй вбить в поиск "chrome.exe" и вставить путь до файла в commands/browser/ahk/run yotube.ahk вместо того пути который там есть

**Comment by RuVl** [comment]
Да, точно, он лежит в program files, а не в program files (x86). Но тогда почему Джарвис умеет открывать хром..?

**Comment by Priler** [comment]
Ошибка известна.
Исправим в ближайшем обновлении.
